ada/language/ast.py

```diff
@@ -252,12 +252,21 @@ def custom_id_text():
         """
         return String("")
 
-    in_aspect = Property(Not(Self.parents.find(
+    in_prepost = Property(Not(Self.parents.find(
```

Comment:
It's a bit of a shame to write a generic `in_aspect` but not being able to use to implement this one :S

Reply:
`in_prepost` does only one parent chain lookup this way. Also it's 2 lines of code, not exactly complex logic :slightly_smiling_face: